bundle/src/main/java/com/adobe/acs/commons/replication/dispatcher/impl/DispatcherFlusherImpl.java

```diff
@@ -44,12 +45,8 @@
  * ACS AEM Commons - Dispatcher Flusher
  * Service used to issue flush requests to enabled Dispatcher Flush Agents.
  */
-@Component(
-        property = {
-                "service.ranking=-10000"
-        },
-        service = {DispatcherFlusher.class}
-)
+@Component
+@ServiceRanking(-10000)
```

Comment:
@kwin @davidjgonzalez 
I fixed an error where the service ranking was wrongly saved as a String instead of an Integer, making the actual value for OSGi zero

This does make the following test fail though:
com.adobe.acs.commons.it.build.ScrMetadataIT.test
Property service.ranking on component com.adobe.acs.commons.replication.dispatcher.impl.DispatcherFlusherImpl has different types (was: {String}, is: {Integer})

WDYT? can we add an ignore in it

Reply:
please add it to ignored properties temporarily

Reply:
done